app/src/containers/FeedData.js

```diff
@@ -52,11 +58,15 @@ const enhanced = compose(
   })),
   lifecycle({
     componentWillMount() {
-      this.props.data.subscribeToMore({
-        document: subscriptionQuery,
-        updateQuery,
-      });
-    }
+      this.unsubscribe = this.props.data.subscribeToMore(subscribeToMore(this.props.user));
+    },
+    componentWillReceiveProps(nextProps) {
+      const userChanged = this.props.user !== nextProps.user;
+      if (userChanged) {
+        this.unsubscribe();
```

Comment:
initially subscribe to updates in `componentWillMount` but update subscription if user auth status has changed